Jenkinsfile

```diff
@@ -530,7 +530,7 @@ def Build_CK(Map conf=[:]){
                     if ( !params.BUILD_LEGACY_OS && arch_type == 1 ){
                             echo "Run inductor codegen tests"
                             sh """
-                                  pip install --break-system-packages --verbose .
+                                  pip install --target ${env.WORKSPACE} --break-system-packages --verbose .
```

Comment:
A longer-term solution could be installing in a virtual environment (either with system's virtualenv or with uv)

Reply:
Yes, this is going to be tricky. I've added a switch to enable/disable that test. We shall have a separate PR to re-enable it.